src/analyzer/analyzersilence.cpp

```diff
@@ -10,33 +10,58 @@ namespace {
 // verify that the track samples have not changed since the last analysis
 constexpr CSAMPLE kSilenceThreshold = 0.001f; // -60 dB
 // TODO: Change the above line to:
-//constexpr CSAMPLE kSilenceThreshold = db2ratio(-60.0f);
+// constexpr CSAMPLE kSilenceThreshold = db2ratio(-60.0f);
+
+// These are in dBV expressed as Volts RMS (which seems, sensibly,
+// the way Mixxx works).
+// Don't change these to const as they are used only to feed the
+// fade thresholds which are consts below themselves while we work
+// out which one is best, and you'll just be adding to exe size.
+#define N_10DB_FADE_THRESHOLD 0.3162f
+#define N_12DB_FADE_THRESHOLD 0.2511f
+#define N_15DB_FADE_THRESHOLD 0.1778f
+#define N_18DB_FADE_THRESHOLD 0.1259f
+#define N_20DB_FADE_THRESHOLD 0.1f
+#define N_24DB_FADE_THRESHOLD 0.0631f
+#define N_25DB_FADE_THRESHOLD 0.0562f
+#define N_27DB_FADE_THRESHOLD 0.0447f
+#define N_30DB_FADE_THRESHOLD 0.0316f
+#define N_40DB_FADE_THRESHOLD 0.01f
```

Comment:
(My intent with those comments was to have a placeholder for the values while I played around.  Assume those are deleted by the time someone is making a change here.  They really are not necessary now)

But I understand the main point.  It is the same problem as the -60db setting.

There would be a cost if we do a lot of processing every time a track is loaded.   Should we have a Cue value that is simply a version flag associated with every track, which could be checked by a future version to fix any issues?  No cue = 4.x or earlier?

For the user adjustment, I suggest that the next step would be to add some UI elements to make this all visible and allow the user adjust.  We also discussed this.  Happy to work on it as a next step (when I get time).  But I think this might be a future version change, based on any perceived need.

Reply:
As I sent that I realised 'Cue value' may not make sense.  But at least have a flag with the track data that shows a version number.  Or, alternatively have a cueinfo.version setting for each cue (though that smells of data duplication).   After a very quick look at the code base - noting exceptions for things like "Serato's tag data", I realise even this might be problematic.  So this too is also, worthy of a chat discussion.

Reply:
I don't know what changes that have been requested remain incomplete at this stage.